pquery-run.sh

```diff
@@ -1388,24 +1399,36 @@ pquery_test(){
           else
```

Comment:
could you print the seed number?

Reply:
Added a line:
echoit "$CMD" . This will print the seed number